tests/com.espressif.idf.ui.test/src/com/espressif/idf/ui/test/executable/cases/project/NewEspressifIDFProjectTest.java

```diff
@@ -362,7 +362,7 @@ private static void whenProjectCleanUsingContextMenu() throws IOException
 		{
 			ProjectTestOperations.launchCommandUsingContextMenu(projectName, bot, "Project Clean");
 			ProjectTestOperations.joinJobByName(Messages.ProjectCleanCommandHandler_RunningProjectCleanJobName);
-			ProjectTestOperations.waitForProjectClean(bot);
+			ProjectTestOperations.findInConsole(bot, "Espressif IDF Tools Console", "Done");
```

Comment:
@AndriiFilippov generally the when*** methods should not contain any verifications that can cause failures if we look deeper into this findInConsole method it can throw exception if it doesnot find this Done and in reality the script may have successfully executed. You are already verifying the project clean operations in the assertions so this is not required